TTHAnalysis/python/plotter/wzsm/fakerate_muonScaleU.txt

```diff
@@ -0,0 +1,15 @@
+weight: 1.0
```

Comment:
What's the difference between fakerate_muonScaleU.txt and fakerate_muonScaleUp.txt? are they both needed?

Reply:
Main difference seems to be my incredible ability to have typos and my liberal use of the "*txt" shortcut. I'll remove it